src/simcore_service_deployment_agent/git_url_watcher.py

```diff
@@ -335,9 +335,10 @@ async def _pull_repository(repo: GitRepo):
 
 
 async def _clone_and_checkout_repositories(
-    repos: list[GitRepo], aio_stack: AsyncExitStack
+    repos: list[GitRepo], aio_stack: AsyncExitStack, synced_via_tags=bool
```

Comment:
``: bool ``  and not ``=bool``

Reply:
We have to introduce mypy in the CI asap

Reply:
would be nice for these silly typos, yes